### DISCUSSION TITLE
'retries=' should be a param of `Client` too.

### DISCUSSION THREAD

**Discussion by NewUserHa** [encode/httpx#2514] [General] [answered]
say you need to use all 'proxy', 'http2', 'retries': 
Currently, you have to manually set the default tranport via:
```py
client = AsyncClient(transport=AsyncHTTPTransport(http2=True, proxy=httpx.Proxy('proxy url'), retries=3))
client.get...
```
it's hard to use, and the routed proxies still can't use 'retries'.

Therefore I think the `client` class should accept 'reties' param and pass it into both `_init_transport` and `_init_proxy_transport`.

**Comment by zanieb** [comment, marked as answer]
Hi! I've proposed this before and begun work on the changes — I'll push the draft for you: https://github.com/encode/httpx/pull/2517/
